lib/EmitVerilog/EmitVerilog.cpp

```diff
@@ -2328,6 +2353,7 @@ void ModuleEmitter::emitOperation(Operation *op) {
     bool visitSV(sv::AssertOp op) { return emitter.emitStatement(op), true; }
     bool visitSV(sv::AssumeOp op) { return emitter.emitStatement(op), true; }
     bool visitSV(sv::CoverOp op) { return emitter.emitStatement(op), true; }
+    bool visitSV(sv::InterfaceInstanceOp op) { return true; }
```

Comment:
Interfaces don't currently have any name, and one gets generated for them. If we used a `name` attribute like elsewhere, we could support naming the interface instances here.

Reply:
Yep, that was a pretty obvious omission from my PR.